client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/domain-bulk-transfer.tsx

```diff
@@ -0,0 +1,104 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button } from '@wordpress/components';
+import { useI18n } from '@wordpress/react-i18n';
+import { translate } from 'i18n-calypso';
+import ThankYouV2 from 'calypso/components/thank-you-v2';
+import { preventWidows } from 'calypso/lib/formatting';
+import { usePresalesChat } from 'calypso/lib/presales-chat';
+import ThankYouDomainProduct from '../products/domain-product';
+import type { ReceiptPurchase } from 'calypso/state/receipts/types';
+
+interface DomainBulkTransferThankYouProps {
+	purchases: ReceiptPurchase[];
+	currency: string;
+}
+
+export default function DomainBulkTransferThankYou( {
+	purchases,
+	currency,
+}: DomainBulkTransferThankYouProps ) {
+	const { __, _n } = useI18n();
+
+	const handleUserClick = ( destination: string ) => {
+		recordTracksEvent( 'calypso_domain_transfer_complete_click', {
+			destination,
+		} );
+	};
+
+	const headerButtons = (
+		<>
+			<Button
+				href="/setup/domain-transfer"
+				onClick={ () => handleUserClick( '/setup/domain-transfer' ) }
+				className="is-secondary"
+			>
+				{ __( 'Transfer more domains' ) }
+			</Button>
+
+			<Button
+				href="/domains/manage"
+				className="manage-all-domains"
+				onClick={ () => handleUserClick( '/domains/manage' ) }
```

Comment:
```suggestion
				href={ domainManagementRoot() }
				className="manage-all-domains"
				onClick={ () => handleUserClick( domainManagementRoot() ) }
```
`domainManagementRoot` is exported from `client/my-sites/domains/paths.js`

Reply:
Nice! Thanks for pointing to the function :)